ports/stm32/boards/VCC_GND_F407ZG/mpconfigboard.h

```diff
@@ -0,0 +1,171 @@
+#define MICROPY_HW_BOARD_NAME       "VCC-GND STM32F407ZG"
+#define MICROPY_HW_MCU_NAME         "STM32F407ZG"
+#define MICROPY_HW_FLASH_FS_LABEL   "VCCGNDF407ZG"
+
+// 1 = use internal flash (1 MByte)
+// 0 = use onboard SPI flash (512 KByte) Winbond W25X40
+#define MICROPY_HW_ENABLE_INTERNAL_FLASH_STORAGE (1)
```

Comment:
why not enable SPI flash by default?

Reply:
The onboard VG flash is 1MB and the ext SPI flash is only 512KB.
... I picked the bigger one as the default.

Reply:
It'd be nice if they could both be enabled/used, one mounted as /flash and the other /spiflash _or something like that._
Like we can with /flash and /sd.
Only one would be mounted as the PYBOARD volume.

Reply:
I noticed a few occurrences of `bdev.c` in various boards. Perhaps could be moved up out of boards and made generic, configured on the board level with presence of new macros. Standard SPI, QSPI, soft and hard interfaces.

Reply:
The available internal storage won't be 1Mbyte, right?  It'll be something small like 256kbyte.  I think it makes sense to enable external by default here.

> I noticed a few occurrences of `bdev.c` in various boards. Perhaps could be moved up out of boards and made generic, configured on the board level with presence of new macros. Standard SPI, QSPI, soft and hard interfaces

Yes, maybe... we can do that separately.

Reply:
Enabled SPI flash as the default.